manim/cli/render/commands.py

```diff
@@ -103,8 +103,6 @@ def __repr__(self):
             try:
                 renderer = OpenGLRenderer()
                 while True:
-                    scene_classes = scene_classes_from_file(file)
```

Comment:
By always rerunning the same scene this breaks the ability to dynamically choose new scenes from the GUI. https://github.com/ManimCommunity/manim/blob/723e645abce9c86cdf40c98959157e6ff93896a0/manim/gui/gui.py#L29

Reply:
I updated it and tested with the GUI and it allows switching between scenes now